dist/index.bundle.js

```diff
@@ -0,0 +1,254 @@
+// deno-fmt-ignore-file
```

Comment:
Please don't include dist files in the repo

Reply:
Thanks. `dist` files are removed.